src/emrun_prejs.js

```diff
@@ -10,6 +10,9 @@
 // Route URL GET parameters to argc+argv
 if (typeof window == 'object') {
   Module['arguments'] = window.location.search.substr(1).trim().split('&');
+  for (let i = 0; i < Module['arguments'].length; ++i) {
+    Module['arguments'][i] = decodeURI(Module['arguments'][i]);
+  }
```

Comment:
Can we write a test for this I wonder?   I see we have `emrun` tests in test_browser.py, maybe we can add one there?